website/content/en/docs/building-operators/ansible/migration.md

```diff
@@ -23,15 +22,27 @@ part of the process to [Integrating Kubebuilder and Operator SDK][integration-do
 
 ### What is new
 
-Scaffolded projects now use:
+Projects are now scaffold using:
```

Comment:
The previous wording is correct: projects are not scaffolded using kustomize, a Makefile, etc, they _use_ kustomize, a Makefile, etc.
```suggestion
Scaffolded projects now use:
```